app/assets/stylesheets/application.css

```diff
@@ -15,3 +15,49 @@
  *= require foundation_and_overrides
 
  */
+/*= require foundation_and_overrides */
+h1 {
```

Comment:
My only comment is that styles like this, and the `body, ul, a, and form` styles you added in this PR will style ALL `body, ul, a, and form` elements on any page anywhere on your site. Its often better to create page specific styles unless you are certain that you want those kind of elements to look that way anywhere on your site.

Otherwise this is certainly good to go.